ote/src/cljc/ote/util/file.cljc

```diff
@@ -0,0 +1,12 @@
+(ns ote.util.file
```

Comment:
Would there be a case when ui is calling these util functions? 
I think that this file could be in clj folder

Reply:
Added commit 1323685